scripts/build/package.js

```diff
@@ -94,19 +94,26 @@ data.icons.forEach((icon) => {
   icons.push(icon);
 
   const iconObject = iconToObject(icon);
+  const iconExportName = slugToVariableName(icon.slug);
 
   // write the static .js file for the icon
   const jsFilepath = path.resolve(iconsDir, `${filename}.js`);
-  writeJs(jsFilepath, `module.exports=${iconObject};`);
+  const newImportMessage = `use "const { ${iconExportName} } = require('simple-icons/icons');" instead`;
+  const message = JSON.stringify(
+    `This way of importing icons has been deprecated in v6.0.0 and will no longer work from v7.0.0, use ${newImportMessage}`,
```

Comment:
I'm thinking now that it could be really weird for users that could see this message without knowing where is being triggered, especially in big codebases. Maybe we can add an even more descriptive message?

```suggestion
    `Imports like "const ${icon.slug} = require('simple-icons/icons/${icon.slug}');" have been deprecated in v6.0.0 and will no longer work from v7.0.0, use ${newImportMessage}`,
```

This would warn:

```
warn - Imports like "const simpleicons = require('simple-icons/icons/simpleicons');" have been deprecated in v6.0.0 and will no longer work from v7.0.0, use "const { siSimpleicons } = require('simple-icons/icons');" instead
```

Reply:
I was thinking that too, but wasn't sure how to solve it. This looks good!